aggregator/src/app/HealthService.ts

```diff
@@ -0,0 +1,17 @@
+export type ResourceHealth = 'healthy' | 'unhealthy';
+
+type HealthCheckResult = {
+  status: ResourceHealth,
+};
+
+export default class HealthService {
+  public overallHealth: ResourceHealth = 'healthy';
+
+  getHealth(): Promise<HealthCheckResult> {
+    this.overallHealth = 'healthy';
+          
+    return Promise.resolve({
+      status: this.overallHealth,
+    });
+  }
```

Comment:
Not sure why we should store `overallHealth`. Let's compute this every time unless we decide to optimize it in future.

```suggestion
  getHealth(): Promise<HealthCheckResult> {
    return Promise.resolve({ status: 'healthy' });
  }
```